### PR TITLE
Add image description and replace fake images in ImageDiff

### PR DESCRIPTION
**Pull request by garfieldduck**
- replace fake images in ImageDiff with real images
- add description to ImageDiff and Image

### CODE REVIEW

File: src/components/article/Image.js [hunk 1]
```diff
@@ -113,11 +102,22 @@ class Image extends FitwidthMixin(Component) {
...
     return (
-      <div ref="imageBox" style={outerStyle} className={styles.imageBox}>
-        {renderedPlaceHoderImage}
-        {renderedFigure}
-        <noscript dangerouslySetInnerHTML={this._getNoscript(_.get(desktop, 'url', ''), _.get(imageByDevice, 'description', ''))} />
+      <div ref="imageBox" className={styles.imageBox}>
+        <div style={outerStyle}>
+          {renderedPlaceHoderImage}
+          {renderedFigure}
+          <noscript dangerouslySetInnerHTML={this._getNoscript(_.get(desktop, 'url', ''), _.get(imageByDevice, 'description', ''))} />
```
Comment (nickhsine) [inline src/components/article/Image.js]: replace ```_.get(imageByDevice, 'description', '')``` with imageDescription?

Comment (garfieldduck) [inline src/components/article/Image.js]: 👌

File: src/components/article/ImageDiff.js [hunk 2]
```diff
@@ -34,42 +37,80 @@ class ImageDiff extends FitwidthMixin(Component) {
...
+    let { desktop } = imageByDevice0
     let { isMounted, screenType, width, percentage, onhovered } = this.state
-    const height = this._getHeight(width, original, width, width)
+    let { outerWidth, outerHeight } = this.props
+    let boxWidth = outerWidth || width
+    let boxHeight = outerHeight || this._getHeight(boxWidth, desktop, DEFAULT_WIDTH, DEFAULT_HEIGHT)
     let buttonClass = null
+    let renderedFigure0 = null
+    let renderedFigure1 = null
+    let imageDescription = _.get(imageByDevice0, 'description', null)
```
Comment (nickhsine) [inline src/components/article/ImageDiff.js]: use ```imageByDevice1.description``` as fallback?

**Comment by nickhsine** [comment]
👍 💯

**Pull request merged**